packages/mobile_app/lib/screens/registration/screen.dart

```diff
@@ -0,0 +1,226 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:mutable_wrappers/mutable_wrappers.dart';
+import 'package:language/language.dart';
+import 'package:app/theme/app_colors.dart';
+import 'package:app/blocs/authentication.dart';
+import 'package:app/components/userinfo_field.dart';
+import 'package:app/components/userinfo.dart';
+import 'package:app/components/gradient_flatbutton.dart';
+import 'package:app/components/userinfo_picker.dart';
+import 'package:user_model/user_model.dart';
+import 'accept_terms.dart';
+
+class RegistrationScreen extends StatelessWidget {
+  final name = MutableString("");
+  final month = MutableString("");
+  final day = MutableString("");
+  final year = MutableString("");
+  final sex = MutableInteger(0);
+  final indexToSex = {
+    0: lang.notSelectedSex,
+    1: lang.male,
+    2: lang.female,
+    3: lang.other,
+  };
+  final country = MutableString("");
+  final place = MutableString("");
+  final termsAccepted = MutableBool(false);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.transparent,
+      body: SafeArea(
+        child: ListView(
+          shrinkWrap: true,
+          padding: EdgeInsets.only(
+            left: 32,
+            right: 32,
+            bottom: 64,
+            top: 4,
+          ),
+          children: <Widget>[
+            Padding(
+              padding: const EdgeInsets.only(bottom: 8.0),
+              child: Image.asset(
+                'assets/launcher/icon.png',
+                height: 96,
+                width: 96,
+              ),
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                Text(
+                  "Daily ",
+                  style: TextStyle(
+                    color: AppColors.accent,
+                    fontSize: 16,
+                  ),
+                ),
+                Text(
+                  "AI ",
+                  style: TextStyle(
+                    fontSize: 16,
```

Comment:
Can we somehow set a theme for a screen? Or add a widget that has only custom fontsize?

Reply:
Only for screen? No.

Reply:
Add text widget with custom size is overkill, add widget with custom user field is not =)